tests/test_predict.py

```diff
@@ -1,53 +1,60 @@
 import matplotlib.image as mpimg
+from fastai.vision import load_learner
+import gdown
 
 from ..src import predict
 from ..src.transform_data import resize, fastai_image
 
 from pathlib import Path
 import numpy as np
-from fastai.basic_train import Learner
 
-import pytest
 
+def test_predict_segment():
+    """
+    Tests predict_segment function in predict
+    by asserting the output shape and its unique
+    values.
+    """
+    # load model
+    try:
+        model_path = Path(__file__).parents[1]
+        learn = load_learner(model_path, 'stage-2_bs16.pkl')
+    except IOError:
```

Comment:
Same issue as before, better to explicitly check the path exists, rather than relying on IOError.